src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -79,10 +145,50 @@ export default class OrganizationIntegrations extends AsyncComponent {
             <Box px={2} flex="1">
               {t('Integrations')}
             </Box>
+            {this.state.reloading && <StyledLoadingIndicator mini />}
           </PanelHeader>
           <PanelBody>{integrations}</PanelBody>
         </Panel>
       </React.Fragment>
     );
   }
 }
+
+const StyledLoadingIndicator = styled(LoadingIndicator)`
+  position: absolute;
+  right: 7px;
+  top: 50%;
+  transform: translateY(-16px);
+`;
+
+const ProviderName = styled.div`
+  font-weight: bold;
+`;
+
+const ProviderDetails = styled(Flex)`
+  align-items: center;
+  margin-top: 6px;
+  font-size: 0.8em;
+`;
+
+const StatusCircle = styled.span`
```

Comment:
Can we use `CircleIndicator`?

Reply:
We can now that you made the color change :)

Reply:
fb04ee7